examples/usecases/retrieval-with-hyperparameter-optimization.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    customer_id = ['customer_id'] &gt;&gt; nvt.ops.Categorify() &gt;&gt; nvt.ops.AddMetadata(tags=[Tags.USER_ID])
</pre><p> </p><p> Instead of nvt.ops.AddMetadata(...)  can we use <code>TagAsUserID()</code> and <code>TagAsItemID()</code>  ops?</p><p>you can also get rid of <code>nvt.ops</code> if you do <code>from nvtabular import *</code> at the beginning. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
change made